chimerapy/graph.py

```diff
@@ -5,6 +5,9 @@
 
 import numpy as np
 import networkx as nx
+import matplotlib
+
+matplotlib.use("TKAgg")
```

Comment:
When using the networkx `draw_networkx` and `plt.figure`, it would cause a Python fatal error without a proper traceback. The recommend suggestion to solve this was to use a different backend.

Reply:
Made it load and setup `matplotlib` only when the `plot` method is called, to avoid backend conflict with external develop usage.